smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -223,27 +267,40 @@ contract DigitalaxAuction is Context, ReentrancyGuard {
         address winner = highestBid.bidder;
         uint256 winningBid = highestBid.bid;
 
-        // Ensure there is a winner
-        require(winner != address(0), "DigitalaxAuction.resultAuction: No one has bid");
+        // Ensure auction not already resulted
+        require(winningBid >= auction.reservePrice, "DigitalaxAuction.resultAuction: reserve not reached");
 
-        // FIXME check to ensure reserve is met
+        // Ensure there is a winner
+        require(winner != address(0), "DigitalaxAuction.resultAuction: no open bids");
```

Comment:
doesn't the require above ensure this is true?

Reply:
reserve can be zero so dont think so no

Reply:
good point